.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: php
 sudo: false
+dist: trusty
```

Comment:
better to just go trusty on the hhvm build only, to avoid getting really old php versions.